modules/volume/include/inviwo/volume/processors/histogramtodataframe.h

```diff
@@ -98,7 +100,7 @@ struct ProcessorTraits<HistogramToDataFrame<Volume>> {
 namespace detail {
 
 IVW_MODULE_VOLUME_API std::shared_ptr<DataFrame> createDataFrame(
-    const std::vector<Histogram1D>& histograms, HistogramMode mode);
+    const std::vector<Histogram1D>& histograms, HistogramMode mode, bool includeDataRangeValues);
```

Comment:
I don't understand what includeDataRangeValues it supposed do

Reply:
The returned DataFrame contains two colums: bin centers in ValueRange and bin counts. If true, this adds an additional column with the bin locations using the original, unmapped scalar values using DataRange.

Reply:
Will they be different? And if so by how much? Should be less then a bin then or?

Reply:
Is this needed for anything else then our debugging?